wiki/Performance_points/de.md

```diff
@@ -0,0 +1,132 @@
+# Performance-Points
```

Comment:
Regarding @Pasi4K5 's concern: I'm afraid that some articles are not strictly implementing the rule `Zusammengesetzte Fremdwörter`. This is especially the case for terms containing three single words, e.g. *Account-Support-Team* or *osu! Community Meeting* (these are their current most used writing styles).

I don't have a string opinion on this one, we can do it either way. Nonetheless should the writing style be consistent in the whole article.

Reply:
This is not resolved. As it stand's the PR is using the dash in the title and nowhere else. It needs to be consistent. 

My personal opinion is that Performance-Points feels weird and we should either be using `Performance Points` (which from my view could be justified as a quote from a foreign language as per [D39]( https://www.duden.de/sprachwissen/rechtschreibregeln/fremdwoerter#D39)) or `Performance-Punkte`, but I'm fine with whatever is decided as long as it is used consistently throughout this PR.

Reply:
I added hyphens to make it more consistent in the entire PR so it is `Performance-Points` (as per [D41](https://www.duden.de/sprachwissen/rechtschreibregeln/fremdwoerter#D41))

Reply:
is this resolved now?

Reply:
Yes, this is resolved with the latest set of commits.